tests/test_client.py

```diff
@@ -163,3 +163,18 @@ def test_old_client_alias() -> None:
     from prisma import Client, Prisma
 
     assert Client == Prisma
+
```

Comment:
Could you also add tests to ensure that the correct number of seconds is passed to `engine.connect()`.

You could use the `patch_method` test function to do this, here's an example of how to use it: https://github.com/RobertCraigie/prisma-client-py/blob/main/tests/test_http.py#L67-L85

Reply:
aren't we testing the same thing in the following test?

https://github.com/RobertCraigie/prisma-client-py/blob/172b5f1bcbe91fda96027625b623f3ba41cd71e2/tests/test_client.py#L99-L113